sal-service/src/main/java/org/ow2/proactive/sal/service/service/JobService.java

```diff
@@ -168,6 +173,7 @@ private String findTaskByProvidedPort(List<TaskDefinition> tasks, String provide
             if (taskProvidesPort(task, providedPortName))
                 return task.getName();
         }
+        LOGGER.error("Task that provides port " + providedPortName + " was not found in job.");
```

Comment:
you should add the job id to the error message

Reply:
Ok.